src/main/java/socialite/logic/commands/AddCommand.java

```diff
@@ -30,18 +32,21 @@
             + "[" + PREFIX_TAG + "TAG]... "
             + "[" + PREFIX_FACEBOOK + "FACEBOOK] "
             + "[" + PREFIX_INSTAGRAM + "INSTAGRAM] "
-            + "[" + PREFIX_TELEGRAM + "TELEGRAM]\n"
+            + "[" + PREFIX_TELEGRAM + "TELEGRAM]"
+            + "[" + PREFIX_TIKTOK + "TIKTOK"
+            + "[" + PREFIX_TWITTER + "TWITTER\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
             + PREFIX_PHONE + "98765432 "
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
-            + PREFIX_TWITTER + "@johndoe"
             + PREFIX_TAG + "friends "
             + PREFIX_TAG + "owesMoney "
             + PREFIX_FACEBOOK + "john.doe"
             + PREFIX_INSTAGRAM + "john.doe"
-            + PREFIX_TELEGRAM + "johndoe";
+            + PREFIX_TELEGRAM + "johndoe"
+            + PREFIX_TIKTOK + "@johndoe"
```

Comment:
Actually, should we remove the `@` symbols here? (since this is an example for the usage of command)